cmd/init.go

```diff
@@ -59,6 +59,14 @@ dapr init -s
 	Run: func(cmd *cobra.Command, args []string) {
 		print.PendingStatusEvent(os.Stdout, "Making the jump to hyperspace...")
 
+		if runtimeVersion == "latest" {
```

Comment:
There can be a different behavior where "latest" is the actual value of the `runtime-version` param and then the logic will override it. Instead, can you change the default to be the environment variable or "latest" if not set, and then, use as the default. So, I would move this change to `init()` in the following way:

```
defaultRuntimeVersion := "latest"
runtimeVersionEnv := viper.GetString("runtime_version")
if runtimeVersionEnv != "" {
  defaultRuntimeVersion = runtimeVersionEnv
}
```

Then, use it in the param.
`InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", defaultRuntimeVersion, "The version of the Dapr runtime to install, for example: 1.0.0")`

Reply:
That's a very subtle difference but I see what you are saying.

Reply:
Addressed the feedback. Thanks!